providers/base/bin/check-ubuntu-desktop-recommends.sh

```diff
@@ -2,20 +2,22 @@
 set -euo pipefail
 IFS=$'\n\t'
 
+target_package=${1}
```

Comment:
Since the default package to test is `ubuntu-desktop`, I would use that as a default in the bash variable. This way, you don't need to modify the job definition. Something like:

```
target_package=${1:-ubuntu-desktop}
```